src/assertions/there.js

```diff
@@ -0,0 +1,21 @@
+import elementExists from '../util/element-exists';
+
+export default function there(client, chai, utils) {
```

Comment:
You can rename it to exists, chai plugins can extend existing assertions.
If you have chai-webdriverio installed the chain should look like

```
expect(testObject.test).to.exist; =>  chaiWebdriverio.exists() => fails => built-in.exists() => passes ===> test passed
```